package EjerciciosInter;

public class Circulo implements Calculable{
    
    private double radio;

    public Circulo(double radio) {

        this.radio = radio;

    }

    @Override 
    public double calcularArea() {
        return Math.PI * Math.pow(radio, 2);
    }

}
